common/profile-model.js

```diff
@@ -37,7 +37,6 @@ export default ({ Meteor, Mongo, LinkableModel, LinkParent, ServerTime }) => {
             index: 1,
             unique: true,
             optional: true,
-            denyUpdate: true,
```

Comment:
The issue here would be that this would allow any user to modify their username at will. A solution to this might be to use `custom: SimpleSchema.denyUntrusted`. Can you give this a try when you get a chance?

Reply:
That worked too!